azure/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "@microsoft/vscode-azext-azureutils",
     "author": "Microsoft Corporation",
-    "version": "0.3.8",
+    "version": "0.3.9",
```

Comment:
These changes may warrant a minor version bump. Specifically around the client/pipeline stuff.

Part of me wants to release a 1.0.0 of the tools package so we can actually use major/minor/patch instead of just minor/patch.

Reply:
I wouldn't mind. We don't really have a "GA" process for these packages so we could just go ahead and do it.